buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -85,7 +85,7 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "exec $(PB_DockerContainerName) $(PB_GitDirectory)/build.sh $(BuildArguments) $(portableLinux) --env-vars \"DISABLE_CROSSGEN=1,TARGETPLATFORM=$(PB_Architecture),TARGETRID=$(TargetRid),PUBLISH_TO_AZURE_BLOB=$(PUBLISH_TO_AZURE_BLOB)\"",
+        "arguments": "exec -e PUBLISH_TO_AZURE_BLOB=$(PUBLISH_TO_AZURE_BLOB) -e CONNECTION_STRING=$(CONNECTION_STRING) -e CLI_NUGET_FEED_URL=$(CLI_NUGET_FEED_URL) -e CLI_NUGET_API_KEY=$(CLI_NUGET_API_KEY) -e NUGET_FEED_URL=$(NUGET_FEED_URL) -e NUGET_API_KEY=$(NUGET_API_KEY) -e GITHUB_PASSWORD=$(GITHUB_PASSWORD) $(PB_DockerContainerName) $(PB_GitDirectory)/build.sh $(BuildArguments) $(portableLinux) --env-vars \"DISABLE_CROSSGEN=1,TARGETPLATFORM=$(PB_Architecture),TARGETRID=$(TargetRid)\"",
```

Comment:
You don't need to use

`-e PUBLISH_TO_AZURE_BLOB=$(PUBLISH_TO_AZURE_BLOB) -e CONNECTION_STRING=$(CONNECTION_STRING)`

Instead you can just use

`-e PUBLISH_TO_AZURE_BLOB -e CONNECTION_STRING`

If you don't specify the value, docker takes the current value of that environment variable, which is what you want.